canonical-web-and-design/vanilla-framework
# Position contextual menu using JS

## Done

Removes any margin compensation code from contextual menu styles.
Updates contextual menu script to position dropdown correctly regardless of the margin.
Removes unnecessary light example.

Fixes #2816

## QA

- Pull code
- Run `./run`
- Open http://0.0.0.0:8101/
- Check contextual menu examples to see if position of contextual menu is correct


## Screenshots

<img width="665" alt="Screenshot 2020-02-12 at 16 53 50" src="https://user-images.githubusercontent.com/83575/74352061-44b11a80-4db8-11ea-90bc-9802821374c1.png">

<img width="661" alt="Screenshot 2020-02-12 at 16 53 42" src="https://user-images.githubusercontent.com/83575/74352066-47137480-4db8-11ea-9c12-acedda8ad8c9.png">
